[0.014s] Invoking command in '/home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/build/chapt4_interfaces': /usr/bin/cmake /home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/chapt4_interfaces -DCMAKE_INSTALL_PREFIX=/home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/install/chapt4_interfaces
[0.140s] -- The C compiler identification is GNU 11.4.0
[0.226s] -- The CXX compiler identification is GNU 11.4.0
[0.240s] -- Detecting C compiler ABI info
[0.303s] -- Detecting C compiler ABI info - done
[0.308s] -- Check for working C compiler: /usr/bin/cc - skipped
[0.309s] -- Detecting C compile features
[0.310s] -- Detecting C compile features - done
[0.314s] -- Detecting CXX compiler ABI info
[0.380s] -- Detecting CXX compiler ABI info - done
[0.385s] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[0.385s] -- Detecting CXX compile features
[0.386s] -- Detecting CXX compile features - done
[0.391s] -- Found ament_cmake: 1.3.12 (/opt/ros/humble/share/ament_cmake/cmake)
[0.511s] -- Found Python3: /usr/bin/python3 (found version "3.10.12") found components: Interpreter 
[0.608s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.614s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.623s] -- Found rosidl_adapter: 3.1.7 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.674s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.716s] -- Found sensor_msgs: 4.9.0 (/opt/ros/humble/share/sensor_msgs/cmake)
[0.901s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.901s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.902s]   chapt4_interfaces --arguments-file
[0.902s]   /home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/build/chapt4_interfaces/rosidl_adapter__arguments__chapt4_interfaces.json
[0.902s]   --output-dir
[0.902s]   /home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/build/chapt4_interfaces/rosidl_adapter/chapt4_interfaces
[0.902s]   --output-file
[0.902s]   /home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/build/chapt4_interfaces/rosidl_adapter/chapt4_interfaces.idls)
[0.902s]   returned error code 1:
[0.902s] 
[0.902s]   Error processing 'int number' of 'chapt4_interfaces/FaceDetector_Response':
[0.902s]   ''int' is an invalid message name.  It should have the pattern
[0.902s]   '^[A-Z][A-Za-z0-9]*$''
[0.902s] 
[0.902s]   Traceback (most recent call last):
[0.902s] 
[0.902s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.902s]       return _run_code(code, main_globals, None,
[0.903s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.903s]       exec(code, run_globals)
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.903s]       sys.exit(main())
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.903s]       abs_idl_file = convert_to_idl(
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.903s]       return convert_srv_to_idl(
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.903s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 863, in parse_service_string
[0.903s]       response_message = parse_message_string(
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.903s]       Type(type_string, context_package_name=pkg_name),
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.903s]       super(Type, self).__init__(
[0.903s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.903s]       raise InvalidResourceName(
[0.903s] 
[0.903s]   rosidl_adapter.parser.InvalidResourceName: 'int' is an invalid message
[0.903s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.904s] 
[0.904s] Call Stack (most recent call first):
[0.904s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.904s]   CMakeLists.txt:12 (rosidl_generate_interfaces)
[0.904s] 
[0.904s] [0m
[0.904s] -- Configuring incomplete, errors occurred!
[0.904s] See also "/home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/build/chapt4_interfaces/CMakeFiles/CMakeOutput.log".
[0.908s] Invoked command in '/home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/build/chapt4_interfaces' returned '1': /usr/bin/cmake /home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/chapt4_interfaces -DCMAKE_INSTALL_PREFIX=/home/bayes/hssw/others/ros2project/products/cpp/service_pratice_ws/src/install/chapt4_interfaces
